openrr-teleop/src/joints.rs

```diff
@@ -127,6 +127,7 @@ where
                 };
             self.joint_trajectory_client
                 .send_joint_positions(pos, self.step_duration)
+                .unwrap()
                 .await
                 .unwrap();
```

Comment:
Below change solve the teleop problem.
I check it macos + urdf-viz, but I believe that it works on Linux/ROS.

```suggestion
            // do not wait
            let _ = self.joint_trajectory_client
                .send_joint_positions(pos, self.step_duration)
                .unwrap();
```